pkg/operator/controllers/pullsecret/pullsecret_controller.go

```diff
@@ -184,15 +184,22 @@ func (r *Reconciler) ensureGlobalPullSecret(ctx context.Context, operatorSecret,
 		// allows for simpler logic flow, when delete and create are not handled separately
 		// this call happens only when there is a need to change, it has no significant impact on performance
 		err := r.client.Delete(ctx, secret)
+		r.log.Info("Global Pull secret Not Found, Creating Again")
 		if err != nil && !kerrors.IsNotFound(err) {
 			return nil, err
 		}
 
 		err = r.client.Create(ctx, secret)
+		if err == nil {
+			r.log.Info("Global Pull secret Created")
+		}
```

Comment:
This might be a matter of preference, but I don't want to have if clause just for one line log.
If you want to check if it failed then you can check outside of the function by checking err returned.

```suggestion
		r.log.Info("Creating global pull secret")
		err = r.client.Create(ctx, secret)	
```

Reply:
I like it the way it is personally, because it only logs when something was successful. If there's a failure, you'd get the error updating the k8s object returned instead. These logs were primarily what we were missing before, but either way works for me.